PythonCodeConventions.md

```diff
@@ -49,6 +58,45 @@ y = fib(2.5)
 In this case, mypy will inform that the call using the floating point value is not valid:
 __mypytest.py:10: error: Argument 1 to "fib" has incompatible type "float"; expected "int"__
 
+The following construct is _only_ allowed in case of circular dependencies, otherwise use regular import statements.
+``` python
+MYPY = False
+if MYPY:
+    from [namespace] import [class]
+```
+Line breaking
+---
+Indentation of function definitions and function calls that exceed the maximum line length (PEP8: 80, Ultimaker: 120 characters) is described in PEP8 https://www.python.org/dev/peps/pep-0008/#indentation. From the suggested set, the following must be applied:
+
+When a function definition exceeds the maximum line length, wrap the lines as such:
+``` python
+def long_function_name(
+  var_one: str,
+  var_two: int,
+  var_three: float,
+  *,
+  var_four: bool=False
+  ) -> None:
+  print(var_one)
+```
+for a function call:
+``` python
+foo = long_function_name(var_one, var_two,
+  var_three, var_four=True)
+```
+When using long named parameters, use a newline for each named parameter:
+``` python
+long_object_name.very_long_function_name(
+    this_is_the_first_variable=var_one,
+    this_is_the_second_variable=var_two
```

Comment:
This is indented inconsistently with the function call and function definition above. Should we indent with 2 spaces or 4?

Reply:
And perhaps we have to write down what indentation to use here, too.

Reply:
Isn't it already agreed upon that we use 4 spaces for each indentation level?
PEP8 describes which indentation to use in the suggested choices.
I did make a mistake with the spacing/indentation in the previous example, which is fixed.

Reply:
As long as it's consistent within the code style standard, that's okay with me. 4 indents it is.